scripts/docker_build.sh

```diff
@@ -21,7 +21,7 @@
 DOCKER_ARGS=$@
 
 GIT_ROOT=$(git rev-parse --show-toplevel)
-GIT_REVISION=$(git describe --abbrev=10 --always --tags --dirty)
+GIT_REVISION=$(git describe --abbrev=10 --always --tags --dirty --match=[0-9]*)
```

Comment:
Interesting, why does this match revisions that start with a letter like "b9e8b63ace"?

Reply:
It's the *name of the tag* being matched, not the revision.